onnxruntime/core/graph/graph_utils.cc

```diff
@@ -289,7 +289,7 @@ bool IsSupportedOptypeVersionAndDomain(const Node& node,
                                        const std::string& op_type,
                                        const std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion>& versions,
                                        const std::string& domain) {
-  return (node.OpType() == op_type && !node.Op()->Deprecated() &&
+  return (node.OpType() == op_type && node.Op() && !node.Op()->Deprecated() &&
```

Comment:
Did you hit a case where an op doesn't have a OpSchema? 
Is that still a valid model? I wonder whether it should error out and crash the program?

Reply:
I am getting error on windows test, it has memory access violation due to this node.Op() is null. It's not an issue on linux test somehow. It's the same model "bert_toy_optimized.onnx" loaded by a few UTs, but failed only in one UT - GradientGraphBuilderTest.TrainingSession_BertToy.